webapp/store/views.py

```diff
@@ -17,7 +17,7 @@
 from urllib.parse import quote_plus
 
 
-def store_blueprint(store_query=None):
+def store_blueprint(app, store_query=None):
```

Comment:
if you need to get some configuration you can:
```
conf = flask.current_app.config.get('CONFIG')
``

This can avoid weird behaviours (circular imports) and it keeps the blueprint "independent" from the main app

Reply:
I was trying to find something like this, thanks